.babelrc

```diff
@@ -0,0 +1,9 @@
+{
+  "presets": [
+    "es2015",
+    "stage-0"
```

Comment:
@reconbot it gets only used for dev it has nothing to do with the production build stage-0 means we can simply use all stuff it makes no sense for dev to add them indipendent dev overhead is nothing we should care of?